tests/phpunit/elementor/core/utils/test-wp-import.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Core\Utils;
+
+use Elementor\Core\Utils\WP_Import;
+use Elementor\Testing\Elementor_Test_Base;
+
+class Test_WP_Import extends Elementor_Test_Base {
+	public function test_run() {
+		// Arrange.
+		$importer = new WP_Import( __DIR__ . '/mock/fresh-install.WordPress.2021-02-14.xml' );
+
+		// Act.
+		$output = $importer->run();
```

Comment:
```suggestion
		$summary = $importer->run();
```
The summary can be:
`[ 'status' => 'success', 'posts' => 3 ]`
or
`[ 'status' => 'failed', 'error' => 'Can't read the XML' ]`
or
`[ 'status' => 'failed', 'error' => 'Nothing to impot' ]`
etc.

Reply:
Handled.